apps/iframe/.env

```diff
@@ -0,0 +1,2 @@
+# react-scripts relies on an older babel than what is installed above, this is to suppress that error.
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
There are a bunch of .DS_Store files added to the repo, these should probably be removed as part of your global gitignore config, cf https://gist.github.com/subfuzion/db7f57fff2fb6998a16c